integration/test/performancetest/performance_test.go

```diff
@@ -42,18 +55,64 @@ func TestPerformance(t *testing.T) {
 		t.Fatalf("Error: %v", err)
 	}
 
-	//------Placeholder to put data into database------//
-	//useless code so data get used and compiler isn't mad
 	if data == nil {
 		t.Fatalf("No data")
 	}
-	//data base 
-	dynamoDB := InitializeTransmitterAPI(DynamoDBDataBase) //add cwa version here
-	if dynamoDB == nil{
-		t.Fatalf("Error: generating dynamo table")
-	}
+	
 	_, err = dynamoDB.SendItem(data)
-	if err !=nil{
+	if err != nil {
 		t.Fatalf("Error: couldnt upload metric data to table")
 	}
-}
\ No newline at end of file
+}
+
+func startLogWrite(t *testing.T, filePath1, filePath2 string, agentRunDuration time.Duration, tps int) {
```

Comment:
I would not do that either. The problem is that this is not scalable so it doesn't account for a variable number of log files that we want to write to. 

What we want to really test out is how cpu and memory change as we add logs to monitor. This makes me think that we should either:
## Decide on a strict path naming convention and then just specify a number of files to write to:
```go
func startLogWrite(... numFiles int) {
    for i := 0; i < numFiles; i++ {
        filepath := fmt.Sprintf("/some/file/path/something%d", i)
    }
}
```

## Pass in a list of file paths:
```go
func startLogWrite(... filePaths []string) {
    for _, fp := range filePaths {
    }
}
```

---
I'm mostly indifferent on which approach to take. They're pretty similar. 

This is also important for adding to the wait group, because we aren't always going to want to monitor two files.

Reply:
I'm fine with both approach. The reason why I let `filePath := []string{"filepath1", "filepath2"}` in the function was:
* Only the performance is using it. Therefore, would consider tuning in the function itself (and it would indeed scalable in its function)
* We can also replace the `filePath := []string{"filepath1", "filepath2"}` to `numberOfFiles` so its easier in adding a files